### PR TITLE
[14.0][MIG] - migration product_brand

### PR DESCRIPTION
**Pull request by sbejaoui**

### CODE REVIEW

File: product_brand/reports/sale_report.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+class SaleReport(models.Model):
+    _inherit = "sale.report"
+
+    product_brand_id = fields.Many2one(comodel_name="product.brand", string="Brand")
+
+    # pylint:disable=dangerous-default-value
+    def _query(
+        self, with_clause="", fields={}, groupby="", from_clause=""  # noqa: B006
+    ):
```
Comment (joao-p-marques) [inline product_brand/reports/sale_report.py:15]: You shouldn't (and don't need to) use this here. The Python standard would be:
```suggestion
    def _query(
        self, with_clause="", fields=None, groupby="", from_clause=""
    ):
        fields = {} or fields
```

File: product_brand/views/product_brand_view.xml [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+        domain="[('product_brand_id', '=', active_id)]"
+    />
+    <act_window
+        id="action_open_single_product_brand"
+        name="Product Brand"
+        res_model="product.brand"
+        view_mode="kanban,form,tree"
+        target="current"
+        domain="[('product_ids', 'in', active_id)]"
+    />
```
Comment (joao-p-marques) [inline product_brand/views/product_brand_view.xml:27]: The `act_window` has been deprecated. You should migrate to the `record` tag. You can use [this](https://github.com/OCA/product-attribute/pull/742/commits/40eca7185f6a8cfb02d3dde10a841a84f92c9178#diff-68f52c9b5cbb5cad326ecb91bf58f75aa8b5a47d4b82f88b0e1dd3b575ea6df7L96-R141) as a reference.

**Review by joao-p-marques — COMMENTED** [review]
I think there are some changes missing for this to be ready for v14.

**Comment by pedrobaeza** [comment]
Superseded by #58

**Pull request closed without merging**